src/common/translation.cpp

```diff
@@ -96,11 +92,23 @@ wxStringToStringHashMap gs_msgIdCharset;
 
 #if wxUSE_LOG_TRACE
 
-void LogTraceArray(const char *prefix, const wxArrayString& arr)
+void LogTraceArray(const char* prefix, const wxArrayString& arr)
 {
     wxLogTrace(TRACE_I18N, "%s: [%s]", prefix, wxJoin(arr, ','));
 }
 
+void LogTraceArray(const char *prefix, const wxVector<wxString>& arr)
+{
+    wxString s;
+    for (wxVector<wxString>::const_iterator j = arr.begin(); j != arr.end(); ++j)
+    {
+        if (j != arr.begin())
+            s += ",";
+        s += *j;
+    }
+    wxLogTrace(TRACE_I18N, "%s: [%s]", prefix, s);
+}
```

Comment:
This is completely unrelated, but I wonder if we shouldn't add `wxJoin()` overload taking `wxVector<wxString>` instead, it looks like it could be useful.

And/or maybe change `wxJoin` to take `wxArrayStringsAdapter`?

Reply:
> This is completely unrelated, but I wonder if we shouldn't add `wxJoin()` overload taking `wxVector<wxString>` instead, it looks like it could be useful.

Implementing an additional `LogTraceArray` overload was the quickest way to get things compiled. Most likely it would be better to add a `wxJoin` overload.

> And/or maybe change `wxJoin` to take `wxArrayStringsAdapter`?

Sorry, never heard about nor used this class. It seems not to be documented.

Reply:
Sorry, this was mostly a note for myself, I'll check if I can do this later, this is not meant to prevent merging of this PR.